sonar-project.properties

```diff
@@ -4,7 +4,7 @@ sonar.organization=telefonicatc2tech
 
 sonar.go.coverage.reportPaths=coverage.txt
 sonar.go.golangci-lint.reportPaths=golangci-report.xml
-sonar.coverage.exclusions=**/*context.go,**/*logger.go,**/*steps.go, **/redis/client.go, **/rabit/amqp.go 
+sonar.coverage.exclusions=**/*context.go,**/*logger.go,**/*steps.go, **/redis/client.go, **/rabit/amqp.go, **/elasticsearch/client.go  
```

Comment:
Group pattern `**/*client.go` because now is a "convention" for the framework 👍🏽